artifactory/commands/transferfiles/state/runstatus.go

```diff
@@ -33,11 +33,12 @@ type TransferRunStatus struct {
 	Version        int    `json:"version,omitempty"`
 	CurrentRepoKey string `json:"current_repo,omitempty"`
 	// True if currently transferring a build info repository.
-	BuildInfoRepo         bool `json:"build_info_repo,omitempty"`
-	CurrentRepoPhase      int  `json:"current_repo_phase,omitempty"`
-	WorkingThreads        int  `json:"working_threads,omitempty"`
-	DelayedFiles          uint `json:"delayed_files,omitempty"`
-	TransferFailures      uint `json:"transfer_failures,omitempty"`
+	BuildInfoRepo         bool   `json:"build_info_repo,omitempty"`
+	CurrentRepoPhase      int    `json:"current_repo_phase,omitempty"`
+	WorkingThreads        int    `json:"working_threads,omitempty"`
+	VisitedDirectories    uint64 `json:"visited_dirs,omitempty"`
```

Comment:
Since Artifactory uses the term "folder" and not "directory", let's make the following change, and implement it anywhwre, including func names and in the UI - 

VisitedFolders    uint64 `json:"visited_folders,omitempty"`